core-tests/shared/src/test/scala/zio/stm/TMapSpec.scala

```diff
@@ -336,6 +337,22 @@ object TMapSpec extends ZIOBaseSpec {
 
         assertM(tx.commit)(equalTo(0))
       }
+    ),
+    suite("folds")(
```

Comment:
```suggestion
    suite("bug #4648")(
```

I'd leave a reference for the future, plus there are folds among operations :).